docs/contrib/README.md

```diff
@@ -8,9 +8,6 @@ low-level access to it's endpoints, when necessary.
 
 It uses the following Javascript technologies and principles:
 
-- [stampit] provides composable Factories based on the [Stamp
-  Specification]. This is how aepp-sdk approached the [composition over
-  inheritance] principle.
```

Comment:
I don't think that we need to keep previously used technologies, someone can check git history if he need